Retarus/main.py

```diff
@@ -0,0 +1,8 @@
+from sekoia_automation.module import Module
+
+from retarus_modules.connector import RetarusConnector
+
+if __name__ == "__main__":
+    module = Module()
+    module.register(RetarusConnector, "retarus_connector")
```

Comment:
The second parameter should be consistent with the trigger manifest. In the trigger manifest you have `retarus_events_trigger` and here `retarus_connector`

Reply:
You mean the `docker_parameters` field? But it seems unrelated